README.md

```diff
@@ -33,7 +33,7 @@ Keeping it simple and with academia in mind, the most fundamentally important as
 
 Now we may ask ourselves: _"What is the ideal programming language?"_
 
-It is the one that is the best in all of the aforementioned fundamental aspects. This language does not exist as pros-and-cons are an integral part of the real world. Therefore, the best _real_ language is the one that overall accumulates the highest quality when explored across these four fundamental aspects.
+It is the one that is the best in all of the aforementioned fundamental aspects. This language does not exist as pros-and-cons are an integral part of the real world. Therefore, the best _real_ language is the one that overall accumulates the highest quality when explored across these five fundamental aspects.
```

Comment:
Could you just remove the count? instead of saying five, just not add a word!

Reply:
Sorry, fixed in [3650686](https://github.com/Datseris/whyjulia-manifesto/pull/6/commits/36506864ea91ffd53a194f233077443c695966fa).